avm/res/web/static-site/main.bicep

```
@@ -88,6 +88,9 @@ param functionAppSettings object = {}
 @description('Optional. The custom domains associated with this static site. The deployment will fail as long as the validation records are not present.')
 param customDomains array = []
 
+@description('Optional. The public network access settings for the static site. `Disabled` is configured by default.')
+param publicNetworkAccess string = 'Disabled'
```

```suggestion
@allowed([
  ''
  'Enabled'
  'Disabled'
])
param publicNetworkAccess string = 'Disabled'
```

Requires a regeneration

---

applied